src/solvers/prop/bdd_expr.h

```diff
@@ -34,6 +34,9 @@ bddt bdd_from_expr(
   std::unordered_map<exprt, bddt, irep_hash> &expr_to_var,
   std::vector<exprt> &var_to_expr);
 
+/// Convert a BDD node to an expression.
+exprt bdd_as_expr(const bdd_nodet &node, const std::vector<exprt> &var_to_expr);
```

Comment:
Just as with `bdd_from_expr`: I still don't understand why this is the right thing to do. It exposes an implementation detail.